Updated MySQL to show setting is deprecated (#8137)
magento/devdocs: src/guides/v2.3/install-gde/prereq/mysql.md

   If this setting is not enabled, `setup:db:status` will always report that `Declarative Schema is not up to date`.

{:.bs-callout-info}

> `explicit_defaults_for_timestamp` is itself deprecated because its only purpose is to permit control over deprecated TIMESTAMP behaviors that are to be removed in a future MySQL release. When removal of those behaviors occurs, `explicit_defaults_for_timestamp` will have no purpose and will be removed as well.

Reviewer: ```suggestion
The `explicit_defaults_for_timestamp` setting is deprecated. This purpose of this setting is to permit control over deprecated TIMESTAMP behaviors that will be removed in a future MySQL release. When those behaviors are removed, the `explicit_defaults_for_timestamp` setting will be removed as well.
```

{:.bs-callout-warning}

> On Magento Cloud Commerce (MariaDB), the setting for `explicit_defaults_for_timestamp` defaults to *OFF*

Reviewer: ```suggestion
On Magento projects deployed on the Cloud platform, the `explicit_defaults_for_timestamp` setting for MySQL (MariaDB) defaults to *OFF*
```
